pkg/iac-providers/kubernetes/v1/load-dir.go

```diff
@@ -43,6 +43,24 @@ func (k *K8sV1) LoadIacDir(absRootDir string) (output.AllResourceConfigs, error)
 			}
 
 			for key := range configData {
+				// the source path formed for each resources is absolute, which should be relative
+				resourceConfigs := configData[key]
+				for i := range resourceConfigs {
+					r := &resourceConfigs[i]
+
+					oldSource := r.Source
+
+					// update the source path
+					r.Source, err = filepath.Rel(absRootDir, r.Source)
+
+					// though this error should never occur, but if occurs for some reason, assign the old value source
+					if err != nil {
+						r.Source = oldSource
+						zap.S().Debug("error while getting the relative path of file", zap.String("IAC file", file), zap.Error(err))
+						continue
+					}
+				}
```

Comment:
Just a suggestion, it would be nice if could move this piece of code into a separate function. then we may be able to unit test as well :)

Reply:
done!